stage1/rootfs/usr_from_coreos/manifest.d/systemd

```diff
@@ -20,7 +20,7 @@ bin/systemd-sysusers
 bin/systemd-tmpfiles
```

Comment:
For now it's a manual process of running ldd on the binaries and debugging. It doesn't take much time and we don't do it often.